src/main/scala/org/ergoplatform/http/api/WalletApiRoute.scala

```diff
@@ -53,6 +55,8 @@ case class WalletApiRoute(readersHolder: ActorRef, nodeViewActorRef: ActorRef, e
         deriveNextKeyR ~
         updateChangeAddressR
     }
+  } ~ pathPrefix("wallet") {
+    signTransactionR
```

Comment:
Shouldn't this be under `toStrictEntity`?  I ran into a strange error, "Substream Source cannot be materialized more than once", but moving `signTransactionR` to be inside `toStrictEntity` seemed to fix the issue.

Reply:
I'm not sure what the error means or why it was happening (maybe JSON data over a particular size?)

Reply:
Also, just noticed that this should be protected by `withAuth` too.

Reply:
now /sign is protected, what and moved under toStrictEntity